### PR TITLE
closes #1041: UI changes to support lbaasV2

### PR DESCRIPTION
**Pull request by kmarquardsen**
model updates
added listeners and security groups

### CODE REVIEW

File: app/scripts/modules/openstack/loadBalancer/details/details.html [hunk 1]
```diff
@@ -54,35 +54,29 @@ <h3 select-on-dbl-click>
...
+        <dd ng-repeat="lbLink in lbLinks track by $index">
           <a href="{{lbLink}}" target="_blank">{{lbLink}}</a>
           <copy-to-clipboard
             class="copy-to-clipboard copy-to-clipboard-sm"
             text="{{lbLink}}"
             tool-tip="Copy Link to clipboard">
           </copy-to-clipboard>
         </dd>
+        <dt>Method</dt>
+        <dd>{{loadBalancer.method}}</dd>
```
Comment (jcwest) [inline app/scripts/modules/openstack/loadBalancer/details/details.html]: Sorry I missed this in the earlier code review... but what is this? And where is the distribution algorithm displayed?

**Comment by jcwest** [comment]
LGTM

**Comment by kmarquardsen** [comment]
@zanthrash @anotherchrisberry I think this work is complete if either of you are ok to merge it

**Comment by kmarquardsen** [comment]
@spinnaker/reviewers PTAL

**Comment by duftler** [comment]
@danielpeach please review.

File: app/scripts/modules/openstack/loadBalancer/transformer.js [hunk 2]
```diff
@@ -33,8 +38,8 @@ module.exports = angular.module('spinnaker.openstack.loadBalancer.transformer',
 
       var healthMonitor = loadBalancer.healthChecks && loadBalancer.healthChecks.length ? loadBalancer.healthChecks[0] : loadBalancer.healthMonitor || {};
```
Comment (danielpeach) [inline app/scripts/modules/openstack/loadBalancer/transformer.js]: How about `_.get(loadBalancer, 'healthChecks[0]') || loadBalancer.healthMonitor || {}`?

**Comment by danielpeach** [comment]
LGTM. One small comment.

**Comment by kmarquardsen** [comment]
@danielpeach updated

**Pull request merged**